Challenge-1/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "forta-bots-challenge",
+  "displayName": "Forta Bots Challenge-1",
+  "version": "0.0.1",
+  "description": "Forta bot that detects creation and updation of other Forta bots on a network",
+  "repository": "https://github.com/yashthipsay/Forta_Bots_Challenges/tree/%40intern-challenges/one",
+  "chainIds": [
+    1,
+    137,
+    10,
+    56,
+    250,
+    42161,
+    43114
+  ],
```

Comment:
Please make sure that only the chains on which the bot is intended to be deployed on are declared here.